### PR TITLE
{chem,math,vis}[foss/2020a,intel/2020a] matplotlib v3.2.1, molmod v1.4.5, yaff v1.6.0 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by ocaisa**
(created using `eb --new-pr`)
requires ~~#10160~~

### CODE REVIEW

**Comment by ocaisa** [comment]
Test report by @ocaisa
**FAILED**
Build succeeded for 2 out of 6 (6 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/7ebd240dffa0ac915271ecc078182880 for a full test report.

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/81b18aaa0e3c0bf7690e03250b52d562 for a full test report.

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/bd765520fdb13226fa82b1db65b47e65 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 8 out of 10 (6 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/aec05e14f77d7c6ff48308c47adb23d5 for a full test report.

**Comment by lexming** [comment]
My test failed due to an outdated `2020a` branch. Re-testing...

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 0 out of 0 (6 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/2a7ae8c7245664ee35206116c81bd6f9 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 7 out of 7 (6 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/746611928eccf3300f6d49b28c03022e for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 1 out of 8 (6 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/ba1a76b50c217ea8b424c13e01961c2d for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
bear-pg0305u03a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/c21999b5ebd0333f4ab71304f76686dc for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0305u03a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/a2c86bb738cd77a7b25764f910a85cde for a full test report.

**Comment by branfosj** [comment]
Building `matplotlib-3.2.1-foss-2020a-Python-3.8.2.eb` - the successful test is with X11 forwarding, the failure is without. This is a change compared to `matplotlib-3.1.1-foss-2019b-Python-3.7.4.eb`.

**Comment by akesandgren** [comment]
Why aren't we using "Agg" instead of TkAgg in the sanity check?? that should eliminatate the problem of X11 forwarding when building?

**Comment by branfosj** [comment]
Either:
```
"""python -c 'import matplotlib; matplotlib.use("Agg"); import matplotlib.pyplot' """
```
or:
```
"""python -c 'import matplotlib; import matplotlib.pyplot' """
```
work for me with no-X11 forwarding.

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/c76207f0b8c48f76051e617ce3fb69f7 for a full test report.

File: easybuild/easyconfigs/m/molmod/molmod-1.4.5_deprecated_time_method.patch [hunk 1]
```diff
@@ -0,0 +1,38 @@
+diff -Nru molmod-1.4.5.orig/molmod/log.py molmod-1.4.5/molmod/log.py
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/m/molmod/molmod-1.4.5_deprecated_time_method.patch]: author and description?

**Comment by ocaisa** [comment]
Test report by @ocaisa
**FAILED**
Build succeeded for 2 out of 6 (6 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/af0f16cb1bfe096cb19e290a04ba32f4 for a full test report.

File: easybuild/easyconfigs/m/molmod/molmod-1.4.5_deprecated_time_method.patch (hunk 1 above)
Comment (ocaisa) [reply easybuild/easyconfigs/m/molmod/molmod-1.4.5_deprecated_time_method.patch]: Done

Comment (ocaisa) [reply easybuild/easyconfigs/m/molmod/molmod-1.4.5_deprecated_time_method.patch]: Updated test reports coming

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/4b20609b7fffc671bba91e1713d01e3f for a full test report.

**Comment by ocaisa** [comment]
Test report by @ocaisa
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/efdb92dfecdcdcd472aebba2186ddae9 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @ocaisa!

**Pull request merged**